ui/static_table/viewmodels/OBATableRow.m

```diff
@@ -23,7 +23,7 @@ + (void)load {
 }
 
 + (instancetype)tableRowWithTitle:(NSString*)title action:(void (^)())action {
-    return [[self alloc] initWithTitle:title action:action];
+    return [[OBATableRow alloc] initWithTitle:title action:action];
```

Comment:
Hmm, actually if you wouldn't mind, include my comment here in the code so that this isn't unintentionally changed in the future :)